docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,32 +1,201 @@
 ---
 sidebar_position: 140
-title: Realtime DB Updates Example
+title: Real-Time DB Updates Example
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Prerequisites from '../../_partials/_prerequisites-sdk-api-key.md';
+import Steps from '../../_partials/_get-started-steps.md';
+import ConnectToGDN from '../../_partials/_connect-to-gdn-code-block.md';
 
-This tutorial is about using Macrometa GDN as a realtime database with local latencies across the globe.
+This tutorial demonstrates how to use Macrometa GDN as a real-time database with local latencies across the globe.
 
-## Prerequisites
+<Prerequisites />
 
-- A Macrometa account with sufficient permissions to create streams.
-- An API key. For more information, refer to [Create API Keys](../../account-management/api-keys/create-api-keys).
-- Appropriate SDK installed. For more information, refer to [Install SDKs](../../sdks/install-sdks.md).
+## Code
```

Comment:
This heading is extremely vague. I suggest something like, "Step-by-Step Instructions"